src/Umbraco.Web.UI.Client/src/packages/ufm/filters/truncate.filter.ts

```diff
@@ -7,8 +7,13 @@ class UmbUfmTruncateFilterApi extends UmbUfmFilterBase {
 		if (tail === 'true') tail = '…';
 		tail = !tail && tail !== '' ? '…' : tail;
 
+		// Only add ellipsis if the string was actually truncated
+		if (str.length <= length) {
+			return str;
+		}
```

Comment:
I think it need to consider `str.trim().length` here.

And possible ready early, so it doesn't need to evaluate `tail`.